lib/oauth2/error.rb

```diff
@@ -16,7 +16,8 @@ def initialize(response)
         message << "#{@code}: #{@description}"
       end
 
-      message << response.body
+      #https://github.com/intridea/oauth2/issues/155
+      message << response.body.force_encoding("UTF-8")
```

Comment:
Sure it shouldn't be `force_encoding('ASCII_8BIT').encode('UTF-8', replace: true)` or something?